### ISSUE TITLE
🐛 Bug Report: Add to favorites icon button not rendered correctly

### ISSUE THREAD

**Issue by mahendra1290** [backstage/backstage#27032]
### 📜 Description

The icon button for Add to favorites under entity is not rendered as a complete circle, and in light theme black color is not visible.

### 👍 Expected behavior

The icon button should render in complete circle and color should be white, before I upgraded to v1.31.3 

<img width="1723" alt="Screenshot 2024-10-09 at 9 57 42 AM" src="https://github.com/user-attachments/assets/4f58de67-54f4-4c27-892b-e528c64df840">

### 👎 Actual Behavior with Screenshots

The icon is rendered in oval shape and also the dark color is not that visible.

<img width="1723" alt="Screenshot 2024-10-09 at 9 51 19 AM" src="https://github.com/user-attachments/assets/13941b74-f5c5-42a2-9258-c2189e6bdccd">

### 👟 Reproduction steps

* Setup a new backstage project with v1.31.3
* Go to any entity page from catalog.
* See the add to favorites icon near entity name.

### 📃 Provide the context for the Bug.

I have recently migrated to v1.31.3 and facing this issue

### 🖥️ Your Environment

```
OS:   Darwin 23.6.0 - darwin/arm64
node: v18.20.3
yarn: 4.4.1
cli:  0.27.1 (installed)
backstage:  1.31.3

Dependencies:
  @backstage/app-defaults                                          1.5.11
  @backstage/backend-app-api                                       1.0.0
  @backstage/backend-common                                        0.24.1, 0.25.0
  @backstage/backend-defaults                                      0.5.0
  @backstage/backend-dev-utils                                     0.1.5
  @backstage/backend-openapi-utils                                 0.1.18
  @backstage/backend-plugin-api                                    0.8.1, 1.0.0
  @backstage/backend-tasks                                         0.6.1
  @backstage/catalog-client                                        1.7.0
  @backstage/catalog-model                                         1.7.0
  @backstage/cli-common                                            0.1.14
  @backstage/cli-node                                              0.2.8
  @backstage/cli                                                   0.27.1
  @backstage/config-loader                                         1.9.1
  @backstage/config                                                1.2.0
  @backstage/core-app-api                                          1.15.0
  @backstage/core-compat-api                                       0.2.8, 0.3.0
  @backstage/core-components                                       0.14.10, 0.15.0
  @backstage/core-plugin-api                                       1.9.3, 1.9.4
  @backstage/dev-utils                                             1.1.1
  @backstage/e2e-test-utils                                        0.1.1
  @backstage/errors                                                1.2.4
  @backstage/eslint-plugin                                         0.1.9
  @backstage/frontend-plugin-api                                   0.7.0, 0.8.0
  @backstage/integration-aws-node                                  0.1.12
  @backstage/integration-react                                     1.1.32
  @backstage/integration                                           1.15.0
  @backstage/plugin-api-docs                                       0.11.10
  @backstage/plugin-app-backend                                    0.3.75
  @backstage/plugin-app-node                                       0.1.25
  @backstage/plugin-auth-backend-module-atlassian-provider         0.3.0
  @backstage/plugin-auth-backend-module-auth0-provider             0.1.0
  @backstage/plugin-auth-backend-module-aws-alb-provider           0.2.0
  @backstage/plugin-auth-backend-module-azure-easyauth-provider    0.2.0
  @backstage/plugin-auth-backend-module-bitbucket-provider         0.2.0
  @backstage/plugin-auth-backend-module-bitbucket-server-provider  0.1.0
  @backstage/plugin-auth-backend-module-cloudflare-access-provider 0.3.0
  @backstage/plugin-auth-backend-module-gcp-iap-provider           0.3.0
  @backstage/plugin-auth-backend-module-github-provider            0.2.0
  @backstage/plugin-auth-backend-module-gitlab-provider            0.2.0
  @backstage/plugin-auth-backend-module-google-provider            0.2.0
  @backstage/plugin-auth-backend-module-microsoft-provider         0.2.0
  @backstage/plugin-auth-backend-module-oauth2-provider            0.3.0
  @backstage/plugin-auth-backend-module-oauth2-proxy-provider      0.2.0
  @backstage/plugin-auth-backend-module-oidc-provider              0.3.0
  @backstage/plugin-auth-backend-module-okta-provider              0.1.0
  @backstage/plugin-auth-backend-module-onelogin-provider          0.2.0
  @backstage/plugin-auth-backend                                   0.23.0
  @backstage/plugin-auth-node                                      0.5.2
  @backstage/plugin-catalog-backend                                1.26.1
  @backstage/plugin-catalog-common                                 1.1.0
  @backstage/plugin-catalog-import                                 0.12.4
  @backstage/plugin-catalog-node                                   1.13.0
  @backstage/plugin-catalog-react                                  1.13.1
  @backstage/plugin-catalog                                        1.23.1
  @backstage/plugin-events-node                                    0.4.0
  @backstage/plugin-home-react                                     0.1.17
  @backstage/plugin-home                                           0.7.11
  @backstage/plugin-org                                            0.6.30
  @backstage/plugin-permission-common                              0.8.1
  @backstage/plugin-permission-node                                0.8.3
  @backstage/plugin-permission-react                               0.4.26
  @backstage/plugin-proxy-backend                                  0.5.6
  @backstage/plugin-scaffolder-common                              1.5.6
  @backstage/plugin-search-backend-module-catalog                  0.2.2
  @backstage/plugin-search-backend-module-pg                       0.5.35
  @backstage/plugin-search-backend-node                            1.3.2
  @backstage/plugin-search-backend                                 1.5.17
  @backstage/plugin-search-common                                  1.2.14
  @backstage/plugin-search-react                                   1.8.0
  @backstage/plugin-search                                         1.4.17
  @backstage/plugin-signals-react                                  0.0.5
  @backstage/plugin-user-settings-common                           0.0.1
  @backstage/plugin-user-settings                                  0.8.13
  @backstage/release-manifests                                     0.0.11
  @backstage/test-utils                                            1.6.0
  @backstage/theme                                                 0.5.6, 0.5.7
  @backstage/types                                                 1.1.1
  @backstage/version-bridge                                        1.0.8, 1.0.9
```

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by stephenglass** [comment]
I believe this was fixed in #26842

**Comment by mahendra1290** [comment]
> I believe this was fixed in #26842 which will be part of the 1.32.0 release

Yes it does thanks closing this issue
